pkg/gatewayserver/io/basicstationlns/basicstationlns.go

```diff
@@ -181,7 +189,19 @@ func (s *srv) handleTraffic(c echo.Context) error {
 		"remote_addr", c.Request().RemoteAddr,
 	))
 
-	ctx, ids, err := s.server.FillGatewayContext(ctx, ttnpb.GatewayIdentifiers{GatewayID: id})
+	// Convert the ID to EUI.
+	str := euiHexPattern.FindAllString(id, -1)
```

Comment:
Why not `FindString`?

Reply:
Same as above. `FindString` exits on the first `e`.

Reply:
If you change the regex as suggested above, this should become `FindStringSubmatch`